pkg/ui/workspaces/cluster-ui/src/store/sagas.ts

```diff
@@ -44,9 +41,6 @@ export function* sagas(cacheInvalidationPeriod?: number): SagaIterator {
     fork(jobsSaga),
     fork(jobSaga),
     fork(databasesListSaga),
```

Comment:
do we still need this saga?

Reply:
Yeah, I kept the ListDatabases stuff around because it seems that's still used elsewhere (sql activity).